bin/hbase

```diff
@@ -508,7 +508,7 @@ add_opentelemetry_agent() {
     fi
     agent_jar=$(tr ':' '\n' < "${f}" | grep opentelemetry-javaagent)
   fi
-  HBASE_OPTS="$HBASE_OPTS -javaagent:$agent_jar"
+  HBASE_OPTS="$HBASE_TRACE_OPTS -javaagent:$agent_jar"
```

Comment:
Good catch!

I don't think this correct either. Shouldn't we be adding `HBASE_TRACE_OPTS` to `HBASE_OPTS`, not overriding `HBASE_OPTS` ?

Reply:
Maybe instead we need an if-block specifically for standalone mode?

Reply:
oops, you're right, I was just testing on standalone mode and forgot we may have more in `HBASE_OPTS`